cmd/ipfs/migration.go

```diff
@@ -82,8 +57,13 @@ func getMigrationFetcher(downloadPolicy string, peers string) (migrations.Fetche
 				return nil, errors.New("custom gateway scheme must be http or https")
 			}
 			fetchers = append(fetchers, migrations.NewHttpFetcher(fetchDistPath, u.String(), httpUserAgent, 0))
+		case "":
+			// Ignore empty string
```

Comment:
Why?

Reply:
Because we state that if they leave the DownloadSources empty, that results in the default behavior.  I thought it might be considered reasonable by some to think `[""]` means empty.